sandbox/TestData2/C.cs

```diff
@@ -1,16 +1,13 @@
 ﻿// Copyright (c) All contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using MessagePack;
-
 #pragma warning disable SA1307 // Accessible fields should begin with upper-case letter
 #pragma warning disable SA1401 // Fields should be private
 
-namespace TestData2
+namespace TestData2;
```

Comment:
In the future, please keep formatting changes in a separate PR. This is adding a lot of noise to a PR that includes functional changes. :)

Reply:
Ok, I will make separate PR which tests C#10 file scoped namespace and global using.